src/enzyme_ad/jax/Passes/EnzymeHLOOpt.cpp

```diff
@@ -18951,6 +18951,20 @@ struct ReduceReduce final : OpRewritePattern<stablehlo::ReduceOp> {
   }
 };
 
+struct ConjReal final : public OpRewritePattern<chlo::ConjOp> {
```

Comment:
did we not have something for this already?

Reply:
the only other conjop pass is ConjComplexNegate